saleor/product/error_codes.py

```diff
@@ -18,3 +18,13 @@ class ProductErrorCode(Enum):
     VARIANT_NO_DIGITAL_CONTENT = "variant_no_digital_content"
     CANNOT_MANAGE_PRODUCT_WITHOUT_VARIANT = "cannot_manage_product_without_variant"
     PRODUCT_NOT_ASSIGNED_TO_CHANNEL = "product_not_assigned_to_channel"
+
+
+class CollectionErrorCode(Enum):
+    DUPLICATED_INPUT_ITEM = "duplicated_input_item"
+    GRAPHQL_ERROR = "graphql_error"
+    INVALID = "invalid"
+    NOT_FOUND = "not_found"
+    REQUIRED = "required"
+    UNIQUE = "unique"
+    CANNOT_MANAGE_PRODUCT_WITHOUT_VARIANT = "cannot_manage_product_without_variant"
```

Comment:
Do you use this error code somewhere? It doesn't sound like an error code for collection.

Reply:
yes, its used in `CollectionAddProducts` which is related to collection errors

Reply:
`collectionAddProducts` can return this error.